src/unix/stream.c

```diff
@@ -1180,6 +1180,9 @@ static void uv__read(uv_stream_t* stream) {
       } else if (errno == ECONNRESET && stream->type == UV_NAMED_PIPE) {
         uv__stream_eof(stream, &buf);
         return;
+#elif defined(_AIX)
+      } else if (errno == ECONNRESET && (stream->flags & UV_DISCONNECT)) {
+        uv__stream_eof(stream, &buf);
```

Comment:
For consistency with the cygwin code block maybe add a `return;`?

Reply:
Return has been added

Reply:
Return has been added.